src/share/classes/com/alibaba/wisp/engine/WispAsyncIO.java

```diff
@@ -0,0 +1,105 @@
+package com.alibaba.wisp.engine;
+
+import java.io.IOException;
+import java.util.concurrent.Callable;
+import java.util.Properties;
+import java.security.AccessController;
+import sun.misc.SharedSecrets;
+import sun.misc.UnsafeAccess;
+import sun.misc.WispAsyncIOAccess;
+import sun.security.action.GetBooleanSecurityPropertyAction;
+import sun.security.action.GetIntegerAction;
+
+public class WispAsyncIO {
+    // AIO
+    static final int CORE_POOL_SIZE;
+    static final int MAX_POOL_SIZE;
+    static final int AIO_QUEUE_LIMIT;
+    static final int TIMEOUT;
+    static final WispAIOSupporter.Policy REJECTED_POLICY;
+    static boolean wisp_aio_done = false;
+
+    static {
+        Properties p = java.security.AccessController.doPrivileged(
+                new java.security.PrivilegedAction<Properties>() {
+                    public Properties run() {
+                        return System.getProperties();
+                    }
+                }
+        );
+        CORE_POOL_SIZE = parsePositiveIntegerParameter(p, "com.alibaba.aio.corePoolSize",
+                Runtime.getRuntime().availableProcessors());
+        MAX_POOL_SIZE = parsePositiveIntegerParameter(p, "com.alibaba.aio.maxPoolSize",
+                Runtime.getRuntime().availableProcessors());
+        AIO_QUEUE_LIMIT = parsePositiveIntegerParameter(p, "com.alibaba.aio.queueLimit", -1);
```

Comment:
I wonder about the magic number. I think we'd better use a named variable to represent the `-1` and reference it here. Or if the `-1` changes afterwards, we may get to change several places all together.